eve/main.yml

```diff
@@ -204,6 +212,7 @@ stages:
             set -ex
             bash wait_for_local_port.bash 8000 40
             npm run ft_test
+          <<: *follow-s3-log
```

Comment:
Does this mean we could see the logs during the build steps instead of waiting until the end?